webpack/scenes/Subscriptions/components/SubscriptionsTable/__tests__/SubscriptionTypeFormatter.test.js

```diff
@@ -0,0 +1,34 @@
+// import React from 'react';
```

Comment:
Sorry for another nitpick, do you mind removing this?

Reply:
That's not a nitpick! Sorry for that, removed.